astropy/units/quantity_helper/function_helpers.py

```diff
@@ -75,7 +75,7 @@
     np.put, np.fill_diagonal, np.tile, np.repeat,
     np.split, np.array_split, np.hsplit, np.vsplit, np.dsplit,
     np.stack, np.column_stack, np.hstack, np.vstack, np.dstack,
-    np.amax, np.amin, np.ptp, np.sum, np.cumsum,
+    np.max, np.min, np.amax, np.amin, np.ptp, np.sum, np.cumsum,
     np.prod, np.product, np.cumprod, np.cumproduct,
     np.round, np.around,
```

Comment:
Could you add `np.round_` here as well? It used to be the same as `np.round` but is now deprecated.

Actually, maybe to help our future selves, add a comment, like
```
    np.round_,  # Alias for np.round in NUMPY_LT_1_25, but deprecated since.
```